lambdas/package/db_utils.py

```diff
@@ -0,0 +1,47 @@
+from package.query_db import query
+from package.lambda_exception import LambdaException
+
+# Written by Ish Chhabra
+
+#Returns whether the user with the given admin_id is admin.
+def is_user_admin(admin_id):
```

Comment:
**_Note - this is an FYI comment! No action required._** 😎 

It's great that these sql queries are off in their own utils file! Generally, the IO operations of an app are considered "side effects" that we want to isolate & test within our applications. A really handy design pattern for SQL queries _(and other IO data access methods)_ is the DAO _(data access object)_ pattern. Basically, we'd create a DAO class / module for each 1st-class object type, that is responsible for accessing & returning the data as an object from the corresponding model. 

Ex - the UsersDAO reads users from mysql, & returns rows as User objects deserialized by the User model.

- /app
  - models
    - User.py
    - Student.py
    - Comment.py
  - daos
    - UsersDAO.py
    - StudentsDAO.py
    - CommentsDAO.py

This is a pattern that can be re-used in many situations! At HubSpot, we use this pattern on our frontend for accessing API data, & on our backend for DB reads. In a statically typed language like java, the models are implemented as interfaces. In loosely typed languages like javascript, we use factory functions, or perhaps a structure from a library such as [ImmutableJS.Record](https://immutable-js.github.io/immutable-js/docs/#/Record).

Reply:
The reason I didn’t felt like doing that was that sometimes we only need to read one column while sometimes we need to read all. So, wouldn’t having a common data reader cause a lot of extra computation?

Reply:
> wouldn’t having a common data reader cause a lot of extra computation and data transfer?

This shouldn't be the case. A DAO is only a slightly more formal abstraction for organizing IO operations.

But this does bring up an interesting concept! There are often tradeoffs between performance optimization, and keeping the overall footprint & complexity of the codebase small. In the case of reading a single db column vs. the whole row, the performance gain is likely very small unless that db row is storing a truly large amount of data. I'm still learning this codebase & app, so if that's the case then, this is a great optimization! If not, then we are taking on extra code that folks have to learn, debug, & maintain, for little return value.

Another benefit of the DAO pattern is that there are very handy tools out there that plug-into DAO methods to make them more expressive & powerful. Ex - [Rosetta](https://github.com/HubSpot/Rosetta)

Reply:
> By the way, this PR is ready to be merged. Right?

That's up to you and your team! Code is very malleable - we can always refactor later if there are higher priorities. I'm still settling in here so my comments won't be requesting changes until I know how things are generally working. 🙂

Reply:
Hi,
We definitely are not storing too much data in the database because for stuff which require too much data (like pictures), we are using s3 bucket. This is something to look into @victoriacaruso.